---
title: "Summarizing FAIR Data Principles"
teaching: 0
exercises: 0
questions:
- "What will your journey to be FAIR productive entail?"
objectives:
- "Discuss steps and changes in your habits you will take after this course"
- "List of resources that will help you achieve these goals"
keypoints:
- "There are simple steps to help make your data more FAIR throughout the research data lifecycle"
- "Implement these steps throughout to keep track of your data and changes"
---

For the past three days we have discussed how you can embark on a journey to be FAIR,
we also showed you tools you can use in your daily life and throughout the research data 
life cycle to make your data more: Findable, Accessible, Interoperable and Reproducible

Simple things like thinking about the metadata that can accompany your data, use of 
templates and preparation of your data for future sharing can help in achieving Open Research
and better FAIR practices. We now want to hear from you what you will change after this
course to be more FAIR:

> ## Exercise 1: Your journey to be FAIRproductive
>
> What are the things you are going to change in your work habits, or actions you are going to take after this course?
>
>> ## Solution
>>
>> *  For each paper I publish I will make a Zenodo deposit with all the data files
>> *  I will use GitHub for all my code instead of using folders
>> *  I will start using ELNs
>> *  Write a data management plan for my project
>> *  Reorganise my folder structure
>> *  Use templates
>> *  Start using readme files (make a readme template)
>> *  Write protocols on protocols.io
>> *  Back up my data accordingly
>> *  Think as a user of my data instead of just a creator
>> *  Start using Jupyter notebooks, or RMarkdown to describe analysis workflows
>> *  Write intelligible metadata 
> {: .solution}
{: .challenge}

As you can see, anything counts. Even simple things like reorganisation of your folder
structure for a better overview or to stop using USB sticks as backup material are steps
towards FAIR. We have collated here the list of resources throughout the course for a 
better overview which will hopefully help you to guide future practices.

> ## Resources
> 
> The BioRDM team has a lot of information about the here taught course material on 
> their [BioRDM wiki](https://www.wiki.ed.ac.uk/display/RDMS/).
> 
> * Details of funding bodies and their involvement and requirements can be found
> at [Plan S/cOAlition S](https://www.coalition-s.org/plan-s-funders-implementation/). 
> * [cOAlition S journal checker tool](https://www.coalition-s.org/blog/unboxing-the-journal-checker-tool/) to assess 
> compliance being developed.
> * The [Directory of Open Access Journals (DOAJ)](https://doaj.org/) is a tool to find which journals are Open Access
> * [Challenges & benefits of OS](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3000246)
> * [Centre for Open Science](https://www.cos.io/)
> * [Ted talk supporting OS](https://www.youtube.com/watch?v=c-bemNZ-IqA)
> * Wiki [Open Science](https://en.wikipedia.org/wiki/Open_science)
> * [European Open Science Cloud](https://www.eosc-hub.eu/open-science-info)
> * [Science is necessarily collaborative - The Biochemist article](https://portlandpress.com/biochemist/article/42/3/58/225220/Science-is-necessarily-collaborative)
> * Digital Object Identifiers [(DOI)](https://www.doi.org/)
> * [ORCID](https://orcid.org/) is a free, unique, persistent identifier that you own and controlâ€”forever
> * Minimum Information for Biological and Biomedical Investigations [(MIBBI)](https://fairsharing.org/collection/MIBBI) is the collection of the most known standards
> * [FAIRSharing](https://fairsharing.org/standards/) offers excellent search service for finding standards
> * [Metadata - How to FAIR](https://howtofair.dk/how-to-fair/metadata/)
> * [Good enough practices in scientific computing (Wilson et al., 2017)](https://doi.org/10.1371/journal.pcbi.1005510)
> * [Good Enough Practices for Scientific Computing](https://github.com/swcarpentry/good-enough-practices-in-scientific-computing)
> * [Organising your data](https://www.data.cam.ac.uk/data-management-guide/organising-your-data)
> * [Organising files and folders](https://www.wur.nl/en/Value-Creation-Cooperation/Collaborating-with-WUR-1/Organising-files-and-folders.htm)
> * [File naming](https://libguides.princeton.edu/c.php?g=102546&p=930626)
> * [Library Carpentry FAIR Data](https://librarycarpentry.org/lc-fair-research/05-reusable/index.html)
> * [Why Jupyter is a data scientists' computational notebook of choice](https://doi.org/10.1038/d41586-018-07196-1)
> * [Coding Standards best practices](https://www.browserstack.com/guide/coding-standards-best-practices)
> * [Reproducible analysis and Research Transparency](https://reproducible-analysis-workshop.readthedocs.io/en/latest/4.Jupyter-Notebook.html)
> * [DMP Online](https://dmponline.dcc.ac.uk/) is a UK tool that is available via subscription to many UK Universities and Institutions
> * [BioRDM wiki DMPs](https://www.wiki.ed.ac.uk/display/RDMS/Data+Management+Plans)
> * [Benefits of writing a DMP - UoE RDS](https://www.ed.ac.uk/information-services/research-support/research-data-service/before/benefits-of-writing-a-dmp)
{: .callout}

> ## Repositories and Ontologies
> #### Data agnostic repositories
> There are general "data agnostic" repositories, for example:
> * [Dryad](http://datadryad.org),
> * [Zenodo](http://zenodo.org),
> * [FigShare](http://figshare.com),
> * [Dataverse](http://thedata.org).
> 
> #### Or domain specific, for example:
> * [UniProt](https://www.uniprot.org/) protein data,
> * [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) sequence data,
> * [MetaboLights](https://www.ebi.ac.uk/metabolights/) metabolomics data
> * [GitHub](https://github.com/) for code.
> 
> Similar to ORCID, there are other registries that can be used to identify many
> of biological concepts and entities:
> * species e.g. [NCBI taxonomy](https://www.ncbi.nlm.nih.gov/Taxonomy)
> * chemicals e.g. [ChEBI](https://www.ebi.ac.uk/chebi)
> * proteins e.g. [UniProt](https://www.uniprot.org/)
> * genes e.g. [GenBank](https://www.ncbi.nlm.nih.gov/genbank/)
> * metabolic reactions, enzymes e.g [KEGG](https://www.genome.jp/kegg/)
> 
> [BioPortal](https://bioportal.bioontology.org/) or
> [NCBI](https://www.ncbi.nlm.nih.gov/)  
> are good places to start searching for a registry or a term.
> 
> #### Here some links to help you find the right ontologies for your research:
> 
> 1. Finding ontologies: [https://bioportal.bioontology.org/](https://bioportal.bioontology.org/)
> 1. List of recommended ontologies: [http://www.obofoundry.org/](http://www.obofoundry.org/)
{: .callout}

> ## Licences we recommend using
> Licenses explicitly declare conditions and terms by which data and software can be re-used.
> Here, we recommend:
> * for data [Creative Commons Attribution (CC BY)](https://creativecommons.org/licenses/by/4.0/)
license
> * for code a permissive open source license such as the [MIT](https://opensource.org/licenses/MIT),
[BSD](https://opensource.org/licenses/BSD-2-Clause),
> or [Apache license](http://www.apache.org/licenses/).
{: .callout}

> ## List of ELNs and online protocols
> * [Benchling](https://www.benchling.com/) 
> * [procols.io](https://www.protocols.io/welcome)
> * Our Benchling tutorial: [BioRDM Wiki](https://www.wiki.ed.ac.uk/display/RDMS/Benchling+%28quick%29+tutorial)
> * [Main help page, with access to several tutorials](https://help.benchling.com/en/)
> * [More molecular biology features](https://help.benchling.com/en/collections/69523-molecular-biology)
> * [Benchling training kit for academics](https://help.benchling.com/en/collections/1608962-benchling-training-kit-for-academics)
> * [ELN Guide](https://www.scinote.net/electronic-lab-notebook-guide/)  
> * comprehensive summary of [ELNs on the University of Edinburgh Wiki](https://www.wiki.ed.ac.uk/pages/viewpage.action?pageId=463750271) by BioRDM
> * [Scinote](https://www.scinote.net/) (free version available)
> * [Rspace](https://www.researchspace.com/) (trial possible)
> * [Labarchives](https://www.labarchives.com/) (trial possible contacting the company)
> * [WikiBench](https://www.ed.ac.uk/information-services/research-support/research-data-service/during/open-research-tools/wikibench) (works on top of Confluence)
{: .callout}

> ## Other useful courses
> You can explore more courses developed by the [Carpentries Incubator](https://github.com/orgs/carpentries-incubator/repositories)
> 
> * [Data Cleaning with OpenRefine for Ecologists](https://datacarpentry.org/OpenRefine-ecology-lesson/)
> * [Open Science with R](https://carpentries-incubator.github.io/open-science-with-r/)
{: .callout}

{% include links.md %}
